code/modules/clothing/clothing.dm

```diff
@@ -331,7 +331,7 @@ BLIND     // can't see anything
 /obj/item/clothing/under/New()
 	if(random_sensor)
 		//make the sensor mode favor higher levels, except coords.
-		sensor_mode = pickweight(list(0 = 1, 1 = 2, 2 = 3, 3 = 2))
+		sensor_mode = pick(0, 1, 1, 2, 2, 2, 3, 3))

```

Comment:
It's pretty garbage readability honestly, it's [number of entries];[the entry]. For absolutely no practical reason I really think the other way around makes more sense.

So for example you could var/x = pick(6;"Hoses!",4;null) and it would functionality equivalent to:
var/x
if(prob(60))
x = "Hoses!"

(but again unless your in a line cutting contest it makes no practical difference)